wordpress-mobile/WordPress-iOS
# Share Extension: Add share extension for the Jetpack app

Part of #19361 

## Description
Creates `JetpackShareExtension` which is a replica of the `WordPressShareExtension`.

~~_PS: This PR still hasn't added the necessary provisioning profiles. This will come in another separate PR. So for now, this can only be tested on simulators and we can not merge this since it will break our builds._~~

P.S: This PR also includes changes in our code signing scripts, merged from #19384 

## Testing Instructions

1. Install Jetpack.
2. Go to the system's Photos app.
3. Select a photo.
4. Tap on the share button.
5. Select Jetpack.
6. Publish the post.
7. Open the Jetpack app.
8. Navigate to posts.
9. Make sure the image is correctly published. 

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

6. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
